.rubocop.yml

```diff
@@ -13,3 +13,12 @@ Style/DocumentationMethod:
 
 Style/ClassAndModuleChildren:
   Enabled: false
+
```

Comment:
I generally follow the Thoughtbot Ruby styleguide which recommends trailing commas

https://github.com/thoughtbot/guides/blob/master/style/ruby/README.md

If you feel strongly however I can change it back